tool/tctl/common/bots_command.go

```diff
@@ -142,6 +144,48 @@ func (c *BotsCommand) ListBots(ctx context.Context, client auth.ClientI) error {
 	return nil
 }
 
+// ListBots writes a listing of the cluster's certificate renewal bots
+// to standard out.
+func (c *BotsCommand) ListBots(ctx context.Context, client auth.ClientI) error {
```

Comment:
The behaviour of `tctl bots ls` effectively changes from listing bot users to listing bots. I don't know of any consumers who are depending on the old behaviour and whilst this is effectively the same, it is a breaking change if someone was depending on fields in the table view having certain values. I don't think the previous info was too useful since it just informed you of the Bot role rather than the roles the bot can impersonate, but thought this was worth highlighting.

Reply:
Could be a bullet point in the changelog